src/animated.tsx

```diff
@@ -61,6 +62,7 @@ export const KeyboardProvider = ({
   statusBarTranslucent,
   navigationBarTranslucent,
 }: KeyboardProviderProps) => {
+  const [enabled, setEnabled] = useState(true);
```

Comment:
Can you add a prop to `KeyboardProvider` that makes `false` the initial state? This will useful for large apps that want to adopt this lib incrementally.

Reply:
Hello @avery-discord 👋 

I can not make it `false` by default, because it'll break a backward compatibility with the previous versions of the library.

However I can add a property to `KeyboardProvider` like `initiallyEnabled` (which will be `true` by default but you can override it) and this piece of the code will be:

```ts
const [enabled, setEnabled] = useState(initiallyEnabled);
```

Will it work for you?

Reply:
Yes, that's exactly what I was thinking. Thank you @kirillzyusko !